packages/skill-lesson/video/collection/index.tsx

```diff
@@ -305,22 +307,46 @@ const Sections = React.forwardRef<SectionsElement, SectionsProps>(
           ref={forwardedRef}
           className={cn('space-y-2', sectionsProps.className)}
         >
-          {module.sections?.map?.((section) => {
-            const childrenWithProps = React.Children.map(children, (child) => {
-              if (React.isValidElement<SectionProps>(child)) {
-                return React.cloneElement(child, {
-                  key: section._id,
-                  section: section,
-                })
-              }
-              return null
-            })
+          {resources.map((resource) => {
+            if (resource._type === 'section') {
+              const childrenWithProps = React.Children.map(
+                children,
+                (child) => {
+                  if (React.isValidElement<SectionProps>(child)) {
+                    return React.cloneElement(child, {
+                      key: resource._id,
+                      section: resource,
+                    })
+                  }
+                  return null
+                },
+              )
 
-            return (
-              childrenWithProps || (
-                <Section key={section._id} section={section} />
+              return (
+                childrenWithProps || (
+                  <Section key={resource._id} section={resource} />
+                )
+              )
+            } else {
+              return (
+                <Primitive.ul
+                  ref={forwardedRef}
+                  // {...lessonsProps }
+                  className={cn(
+                    'bg-background border-x border-b border-card py-2 rounded-b',
+                    // lessonsProps.className,
+                  )}
+                >
+                  <li key={resource._id}>
+                    <Lesson
+                      section={undefined}
+                      lesson={resource}
+                      index={resource._id}
+                    />
+                  </li>
```

Comment:
ideally this could be styled using children approach, similar to the rest. I'll look into it asap.